src/v2/stores/accounts/detail.js

```diff
@@ -45,4 +45,4 @@ decorate(Store, {
 
 const AccountDetailStore = new Store();
 
-export default observable(AccountDetailStore);
```

Comment:
observable can't be used here. observable values can be JS primitives, references, plain objects, class instances, arrays and maps.

Reply:
this makes sense - we should apply it to all the newer stores. If you can do it, great! If not,
I can take a pass through & take care of it...